Cargo.toml

```diff
@@ -8,6 +8,9 @@ edition = "2018"
 clap = "2.33.0"
 lazy_static = "1.4.0"
 log = { version = "0.4.8", features = ["std"] }
+packet-builder = "0.3.0"
+pnet = "0.22.0"
+pnet_base = "0.22.0"
```

Comment:
If they're only needed for test put them in "dev-dependencies" below.

Reply:
ok, it's moved to be under "dev-dependencies".